src/Discord/Internal/Rest/Channel.hs

```diff
@@ -308,6 +308,12 @@ instance ToJSON StartThreadOpts where
       , "rate_limit_per_user" .=? startThreadRateLimit
       ]
 
+instance Default StartThreadOpts where
+  def = StartThreadOpts { startThreadName        = ""
```

Comment:
reading the docs it looks like the thread name must be at least one character; maybe we can add that to the above documentation, as well as set a default thread name of `no thread name` or similar to make sure the default works?

Reply:
maybe the name of `New Thread`? As discord suggests by default
![obraz](https://github.com/discord-haskell/discord-haskell/assets/93288843/82cd5513-8eff-4e8d-b2fd-fd72ccbe26b4)